### PR TITLE
use template for pypi-mobans LICENSE

### PR DESCRIPTION
**Pull request by jayvdb**
I'm not sure these changes are good, but it is a bit odd that this repo doesnt use its own template .. ;-)

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -13,17 +12,19 @@ modification, are permitted provided that the following conditions are met:
   this list of conditions and the following disclaimer in the documentation
   and/or other materials provided with the distribution.
 
-* Neither the name of the copyright holder nor the names of its
-  contributors may be used to endorse or promote products derived from
-  this software without specific prior written permission.
+* Neither the name of 'pypi-mobans' nor the names of the contributors
+  may not be used to endorse or promote products derived from this software
```
Comment (jayvdb) [inline LICENSE]: this `not` is wrong!

File: config/data.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-name: setupmobans
-nick_name: setupmobans
-copyright_year: 2017
+name: pypi-mobans
+nick_name: pypi-mobans
+copyright_year: 2018
```
Comment (jayvdb) [inline config/data.yml]: I guess this should be 2017-2018

File: LICENSE (hunk 1 above)
Comment (chfw) [reply LICENSE]: I think `not` should not be there. double negative is positive.

**Comment by jayvdb** [comment]
Sorry, this shouldnt have been merged

**Pull request merged**